org.civicrm.reqangular/js/test/services/api/contract.spec.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'common/lodash',
+  'common/mocks/data/contract.data',
+  'common/angularMocks',
+  'common/services/api/contract',
+  'common/mocks/services/api/contract-mock'
+], function (_, jobContractData) {
+  'use strict';
+
+  describe('api.contract', function () {
+    var JobContractAPI, $rootScope, $q;
+
+    beforeEach(module('common.apis', 'common.mocks'));
+
+    beforeEach(inject(['api.contract', '$rootScope', '$q',
+      function (_JobContractAPI_, _$rootScope_, _$q_) {
+        JobContractAPI = _JobContractAPI_;
+        $rootScope = _$rootScope_;
+        $q = _$q_;
+      }]));
+
+    describe('getContactsWithContractsInPeriod()', function () {
+      var result;
+      var sampleDates = {
+        start: '2018-01-01',
+        end: '2018-01-31'
+      };
+
+      beforeEach(function (done) {
+        spyOn(JobContractAPI, 'sendGET').and.returnValue(
+          $q.resolve(jobContractData.contactsWithContractsInPeriod));
```

Comment:
```js
spyOn(JobContractAPI, 'sendGET').and.returnValue(
  $q.resolve(jobContractData.contactsWithContractsInPeriod)
);
```

Reply:
@AkA84 I think this one is fine actually. We do this all the time. It does not hurt the readability but it saves a line.